src/store/slices/auth.slice.ts

```diff
@@ -13,9 +13,10 @@ const initialState: AuthState = {
   features: [],
 };
 
-export const fetchActiveSubscription = createAsyncThunk('fetchActiveSubscription', async (payload: {}, thunkApi) => {
+export const fetchActiveSubscription = createAsyncThunk('fetchActiveSubscription', async (payload: any, thunkApi) => {
```

Comment:
We can apply the same principle like the one I suggested for the `createTestRun` action here
Mostly because the `profileId` can be obtained from a redux store state instead of asking it as a function parameter